tests.toml

```diff
@@ -0,0 +1,31 @@
+test_format = 1.0
+
+[default]
+
+    # -------------------------------
+    # Default args to use for install
+    # -------------------------------
+
+    args.with_sftp = "1"
+	args.phpversion = "7.4"
+	args.with_mysql = "1"
+
+[7.3_test]
+
+    only = ["install.subdir", "backup_restore", "upgrade" ]
+    args.phpversion = "7.3"
```

Comment:
Uuuh unfortunately we can't use `.` in the name here, though it acts as if it works, but this is equivalent to write:

```
[7]
    [7.3_test]
            only = ["install.subdir", "backup_restore", "upgrade" ]
```

with `7` being one level, and `3_test` being another level

which is not what we want :sweat_smile: 

Maybe we should rename the test to `php_73` for example or `73_test` or whatever, but without a dot :D